audpsychometric/core/gold_standard.py

```diff
@@ -197,15 +197,15 @@ def mode(
     )
 
 
-def rater_agreement_pearson(
+def rater_agreement(
     ratings: typing.Sequence,
     *,
     axis: int = 1,
 ) -> np.ndarray:
     """Calculate rater agreements.
 
     Calculate the agreement of a rater
-    by the correlation of a rater
+    by the Pearson correlation of a rater
```

Comment:
Would you then not want a keyword argument that allows to paramtrize the association measure - and that defaults to the pearson correlation?.

Reply:
No, I would add the keyword argument only when we provide other options beside Pearson. Which would require adding additional tests as well.

When adding that argument to the API at a later stage, we need then indeed set the default value to Pearson.

Reply:
Alright, this is crossed now. I have approved for now anyway which is probably the best we can do for now.